LiskHQ/lisk
# Update Dependencies not requiring code changes - Closes #2498

### What was the problem?

Some dependencies were out of date and the version jump didn't introduce breaking changes.

### How did I fix it?

Check change logs of dependencies and upgraded the ones with no breaking changes.

### How to test it?

- Run test suit
- Check build passes

### Review checklist

* The PR resolves #2498
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
